wordpress-mobile/gutenberg-mobile
# Merge release 1.55.0 to develop

This PR was created to **Sync the Release to Development Branches**
Internal ref: `p8Qyks-10e-p2`

To test:
Verify that the gutenberg reference points to the merge commit in `gutenberg` trunk of the [gutenberg PR](https://github.com/WordPress/gutenberg/pull/32619)

PR submission checklist:
